temoa/_internal/run_actions.py

```
@@ -202,24 +202,32 @@ def solve_instance(
+    # explicitly (e.g. via the *_solver_options.toml files used by the extensions).
     if solver_name == 'cbc':
         pass
 
     elif solver_name == 'cplex':
-        # Note: these parameter values match mip-dev / PyPSA
-        # (see: https://pypsa-eur.readthedocs.io/en/latest/configuration.html)
-        optimizer.options['lpmethod'] = 4  # barrier
-        optimizer.options['solutiontype'] = 2  # non basic solution, ie no crossover
-        optimizer.options['barrier convergetol'] = 1.0e-3
-        optimizer.options['feasopt tolerance'] = 1.0e-4
+        pass
```

Ruff (0.15.21)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)